Gemfile

```diff
@@ -1,3 +1,9 @@
 source 'https://rubygems.org'
 
+group :development, :test do
+  gem 'simplecov', require: false
+  gem 'rubocop', '~> 0.37.2', require: false unless RUBY_VERSION =~ /^1.8/
```

Comment:
Why is there Ruby version check? There is no support for Ruby 1.8. That should not be there at all.